cron/centAcl.php

```diff
@@ -377,70 +374,62 @@
         /** ***********************************************
          * Get all included Hosts
          */
-        $hostIncCache = array();
+        $hostIncCache = [];
         $dbResult = $pearDB->query(
-            "SELECT host_id, host_name, acl_res_id
-            FROM `host`, acl_resources_host_relations
-            WHERE acl_resources_host_relations.host_host_id = host.host_id
-                AND host.host_register = '1'"
+            "SELECT host_host_id, acl_res_id
+            FROM acl_resources_host_relations"
         );
         while ($h = $dbResult->fetch()) {
             if (!isset($hostIncCache[$h["acl_res_id"]])) {
-                $hostIncCache[$h["acl_res_id"]] = array();
+                $hostIncCache[$h["acl_res_id"]] = [];
             }
-            $hostIncCache[$h["acl_res_id"]][$h["host_id"]] = $h["host_name"];
+            $hostIncCache[$h["acl_res_id"]][$h["host_host_id"]] = 1;
```

Comment:
Why you fix the value "1"?

Reply:
I guess cause isset is better (in performance) than in_array